inception-kb/src/main/java/de/tudarmstadt/ukp/inception/kb/graph/KBStatement.java

```diff
@@ -51,10 +52,41 @@
 
     private boolean inferred;
 
-    private List<Statement> originalStatements = new ArrayList<>();
+    private List<Statement> originalStatements;
+
+    private List<KBQualifier> qualifiers;
```

Comment:
Brauchen wir die Liste wirklich?

Reply:
I don't like the List either. I'll try to not use it for KB front end. If it works, then I'll remove it.